bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -106,8 +107,8 @@ http(s)://xx.yy.zz.ww:443
 
 Your Netatmo App will be configured automatically by the bridge to the endpoint:
 
-```text
-http(s)://xx.yy.zz.ww:443/netatmo/webhook/<_CLIENT_ID_>
+```
```

Comment:
The "text" was there probably for a good reason.
By the way, there is one just above for another HTTP URL. It should be maintained consistent, either remove both or keep both.

Reply:
I don't have that neither